tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -234,11 +234,23 @@ Wait Until JupyterLab Is Loaded
   Wait Until Element Is Visible  xpath:${JL_TABBAR_CONTENT_XPATH}  timeout=${timeout}
 
 Clone Git Repository
+  [Documentation]    Clones git repository and logs error message if fails to clone
   [Arguments]  ${REPO_URL}
-  Navigate Home (Root folder) In JupyterLab Sidebar File Browser
-  Open With JupyterLab Menu  Git  Clone a Repository
-  Input Text  //div[.="Clone a repo"]/../div[contains(@class, "jp-Dialog-body")]//input  ${REPO_URL}
-  Click Element  xpath://div[.="CLONE"]
+  ${status}    ${err_msg} =    Run Keyword and Ignore Error    Clone Repo and Return Error Message    ${REPO_URL}
+    IF    "${status}" == "PASS"
```

Comment:
Shouldn’t it do this is status <> PASS ?

Reply:
I tried it, it is not working